libraries/freertos_plus/standard/freertos_plus_tcp/source/portable/NetworkInterface/Zynq/uncached_memory.c

```diff
@@ -86,31 +42,52 @@
  * uncached memory.
  */
 
+/* Standard includes. */
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
 /* FreeRTOS includes. */
 #include "FreeRTOS.h"
 #include "task.h"
+#include "queue.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_Sockets.h"
+#include "FreeRTOS_IP_Private.h"
 
-/* Xilinx includes. */
 #include "Zynq/x_emacpsif.h"
 #include "Zynq/x_topology.h"
 #include "xstatus.h"
+
 #include "xparameters.h"
 #include "xparameters_ps.h"
 #include "xil_exception.h"
 #include "xil_mmu.h"
 
-/* Dem includes. */
 #include "uncached_memory.h"
```

Comment:
The copy of uncached_memory.{c, h} was outdated compared to the FreeRTOS+TCP release.
I cleaned it further up, mostly style changes.